### PR TITLE
Feature/claims

### PR DESCRIPTION
**Pull request by fillup**

### CODE REVIEW

**Review by jason-jackson — APPROVED** [review]

File: application/api/claims.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package api
+
+import (
+	"time"
+
+	"github.com/gofrs/uuid"
+)
+
+type (
```
Comment (Baggerone) [inline application/api/claims.go:9]: Should these all (and below) go to the api like the other kinds of `Type` and `Status` consts?

File: application/api/claims.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	EventType        ClaimEventType `json:"event_type"`
+	EventDescription string         `json:"event_description"`
+	Status           ClaimStatus    `json:"status"`
+	ReviewDate       time.Time      `json:"review_date,omitempty"`
+	ReviewerID       uuid.UUID      `json:"reviewer_id,omitempty"`
+	PaymentDate      time.Time      `json:"payment_date,omitempty"`
+	TotalPayout      int            `json:"total_payout,omitempty"`
+	CreatedAt        time.Time      `json:"created_at"`
+	UpdatedAt        time.Time      `json:"updated_at"`
+}
```
Comment (Baggerone) [inline application/api/claims.go:43]: Do you want the Policy and Reviewer fields already, or wait on those?

File: application/api/claimitems.go [hunk 3]
```diff
@@ -0,0 +1,36 @@
+package api
+
+import (
+	"time"
+
+	"github.com/gofrs/uuid"
+)
+
+type ClaimItemStatus string
```
Comment (Baggerone) [inline application/api/claimitems.go:9]: Should these all go in the api package?

File: application/models/claimitems.go [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+	}
+
+	if err := c.LoadItem(tx, false); err != nil {
+		domain.ErrLogger.Printf("failed to load item for claim item: %s", err)
+		return false
+	}
+
+	var policy Policy
+	if err := policy.FindByID(tx, c.Item.PolicyID); err != nil {
+		domain.ErrLogger.Printf("failed to load policy for dependent: %s", err)
```
Comment (Baggerone) [inline application/models/claimitems.go:83]: This error message looks like a copy-paste relic that needs converting.

**Review by Baggerone — APPROVED** [review]

**Pull request merged**